src/Tools/dotnet-monitor/DiagnosticsMonitorCommandHandler.cs

```diff
@@ -71,10 +73,10 @@ private static void ConfigureMetricsEndpoint(IConfigurationBuilder builder, stri
         {
             builder.AddInMemoryCollection(new Dictionary<string, string>
             {
-                {MakeKey(nameof(PrometheusConfiguration), nameof(PrometheusConfiguration.Endpoints)), string.Join(';', metricEndpoints)},
-                {MakeKey(nameof(PrometheusConfiguration), nameof(PrometheusConfiguration.Enabled)), true.ToString()},
-                {MakeKey(nameof(PrometheusConfiguration), nameof(PrometheusConfiguration.UpdateIntervalSeconds)), 10.ToString()},
-                {MakeKey(nameof(PrometheusConfiguration), nameof(PrometheusConfiguration.MetricCount)), 3.ToString()}
+                {MakeKey(MetricsOptions.ConfigurationKey, nameof(MetricsOptions.Endpoints)), string.Join(';', metricEndpoints)},
+                {MakeKey(MetricsOptions.ConfigurationKey, nameof(MetricsOptions.Enabled)), true.ToString()},
+                {MakeKey(MetricsOptions.ConfigurationKey, nameof(MetricsOptions.UpdateIntervalSeconds)), 10.ToString()},
+                {MakeKey(MetricsOptions.ConfigurationKey, nameof(MetricsOptions.MetricCount)), 3.ToString()}
```

Comment:
It might be worthwhile if we have any samples, blogs, etc that use the old keys. I know one of my posts on an Issues page uses them.

Reply:
The announcement blog post seems to only make reference to the metrics endpoint and that it outputs Prometheus exposition format, but not what the configuration options are. Looks like the issue you are probably described the previous name in is #1547

@shirhatti, do we want to consider tracking somewhere the changes that we make during previews until dotnet-monitor ships? Maybe a GitHub issue?